lib/diagnostic.rb

```diff
@@ -46,6 +57,10 @@
 
 ##
 # your response here
+Class Developer < Person
+def solve_problems
+  yield "think think think"
```

Comment:
Here, you don't need the `yield` keyword. If we wanted to return, we would need no keyword at all!